contrib/src/main/java/com/datatorrent/contrib/hbase/HBaseWindowStore.java

```diff
@@ -126,14 +134,17 @@ public void connect() throws IOException {
 
   @Override
   public void beginTransaction() {
-    // HBase does support transactions so this method left empty
+    // HBase does not support transactions so this method left empty
```

Comment:
Similar change needed in rollbackTransaction() and isInTransaction() . And possibly others. Pls check everywhere for this comment and fix it.